src/reflection.cpp

```diff
@@ -742,4 +742,13 @@ bool Verify(const reflection::Schema &schema, const reflection::Object &root,
   return VerifyObject(v, schema, root, flatbuffers::GetAnyRoot(buf), true);
 }
 
+bool VerifySizePrefixed(const reflection::Schema &schema,
+                        const reflection::Object &root, const uint8_t *buf,
+                        size_t length, uoffset_t max_depth /*= 64*/,
```

Comment:
all these variables can be marked const

Reply:
You can also remove the /*= 64*/ from here and max_tables. Those will comments will rot.

Reply:
should I change all that for Verify, which I just copied?